modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney/base_controller.rb

```diff
@@ -114,7 +114,7 @@ def set_auth_headers
           if allow_dependent_claimant?
             add_dependent_to_auth_headers(headers)
           else
-            auth_headers
+            headers
```

Comment:
We actually do need the intermediate variable here, as `auth_headers` is actually a method defined in modules/claims_api/app/controllers/claims_api/v2/application_controller.rb. So any time we used that, it would call the method and re-create the headers, wiping any merges done previously to the auth_headers.